### PR TITLE
[CDAP-3355] Fix typo in runtime argument for workflow

### PR DESCRIPTION
**Pull request by bdmogal**

### CODE REVIEW

File: cdap-docs/examples-manual/source/examples/wikipedia-data-pipeline.rst [hunk 1]
```diff
@@ -115,7 +115,7 @@ There are three ways to start the *WikipediaPipelineWorkflow* program:
 
 You can also (optionally) specify these runtime arguments for the *WikipediaPipelineWorkflow*:
 
-- *min.pages.threshold*: Threshold for the number of pages to exist in the *pageTitleStream* for the workflow to proceed.
+- *min.page.threshold*: Threshold for the number of pages to exist in the *pageTitleStream* for the workflow to proceed.
```
Comment (awholegunch) [inline cdap-docs/examples-manual/source/examples/wikipedia-data-pipeline.rst]: Why not change the code instead? It is a plural (number of pages)...

**Comment by anwar6953** [comment]
LGTM

**Comment by awholegunch** [comment]
Though now, you need to update the docs build...let me get you the changes...

**Comment by awholegunch** [comment]
LGTM. Neither of these pages are included in the docs, so we're good.

**Comment by bdmogal** [comment]
Build passed. Merging.

**Pull request merged**